pkg/controller/injector/k8s_binding_injector.go

```diff
@@ -60,7 +60,6 @@ func CreateK8sBindingInjector() (BindingInjector, error) {
 
 func (b *k8sBindingInjector) Inject(binding *servicecatalog.Binding, cred *brokerapi.Credential) error {
 	is := makeInjectionSet(binding, cred)
-
 	if err := b.injectSecret(is.secret); err != nil {
```

Comment:
Net, it seems to me we could simplify this entire file (from here on) down to just two funcs: InjectBinding and UninjectBinding - and all bits of code could be inlined making it much easier to follow (and of course killing injectionSet).